src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -38,13 +39,18 @@ private Configuration() {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Configuration.class);
 
+    private static final String DEFAULT_PERSEO_FE_URL = "http://127.0.0.1:9090";
+    private static final long DEFAULT_MAX_AGE = 60000;
```

Comment:
Maybe 30000 sjhould be used as default (as it is now in master)?

Reply:
OK, I think it's appropriate to change it.

Reviewing your comment I thought it's important to review the changes that this PR causes to minimize possible errors.

Yes, is true that in the original Configuration.java exist a  *DEFAULT_MAX_AGE = 30000* but in fact this value was only used if [perseo-core.properties](https://github.com/telefonicaid/perseo-core/blob/master/src/main/resources/perseo-core.properties) file is deleted.

As I see it, in normal circunstantes, reload function is call from the begining and *maxAge* attribute [take the value of *rule.max_age*](https://github.com/telefonicaid/perseo-core/blob/master/src/main/java/com/telefonica/iot/perseo/Configuration.java#L68) property from [perseo-core.properties](https://github.com/telefonicaid/perseo-core/blob/master/src/main/resources/perseo-core.properties). For this reason, I considered that the real default values for DEFAULT_PERSEO_FE_URL and DEFAULT_MAX_AGE, are the indicated in [perseo-core.properties](https://github.com/telefonicaid/perseo-core/blob/master/src/main/resources/perseo-core.properties) (60000 and http://127.0.0.1:9090).

This PR changes the implementation of [perseo_core-entrypoint.sh](https://github.com/telefonicaid/perseo-core/blob/master/perseo_core-entrypoint.sh), establishing the PERSEO_FE_URL_VALUE as environment var when exists and there are not other PERSEO_FE_URL env var assigned previously. If I am not mistaken, this new way results in the next configuration order priority:

1º- Environment vars
2º- Entrypoint
3º- perseo-core.properties
4º- Default values established in the new Configuration Class

Reply:
Ok, makes change. The `DEFAULT_MAX_AGE = 30000` was never used. The configuration priority is also ok. Actually, 4º will never happen as `DEFAULT_MAX_AGE` is no longer used.

NTC (from my point of view)

Reply:
In that case I'd suggest to have some defaults in the code (DEFAULT_PERSEO_FE_URL and DEFAULT_MAX_AGE) which values match the ones of current perseo-core.properties file. Does it make sense?

Reply:
Yes, I think so. That is the current behavior.

Then, it is not necessary to change anything more.